CHANGELOG.md

```diff
@@ -937,6 +943,10 @@ Please read the [docs]() for more details.
 [#806]: https://github.com/DataDog/puppet-datadog-agent/issues/806
 [#820]: https://github.com/DataDog/puppet-datadog-agent/issues/820
 [#821]: https://github.com/DataDog/puppet-datadog-agent/issues/821
+
+[#824]: https://github.com/DataDog/puppet-datadog-agent/issues/824
+
+[#835]: https://github.com/DataDog/puppet-datadog-agent/issues/835
```

Comment:
should these go under 3.24?

Reply:
Yeah, they're linked in the `3.24.0` changelog entry

Reply:
doh, sorry! i misread (missed the compacted lines). do you mind removing the extra line breaks for consistency?